protocol/grpc/client.go

```diff
@@ -123,3 +134,13 @@ func getInvoker(impl interface{}, conn *grpc.ClientConn) interface{} {
 	res := method.Call(in)
 	return res[0].Interface()
 }
+
+//client timeout
+func UnaryClientTimeoutInterceptor(timeout time.Duration) grpc.UnaryClientInterceptor {
```

Comment:
I think it is private method for `grpc` package, no need to export

Reply:
I'm sorry, let me fix that

Reply:
> I think it is private method for `grpc` package, no need to export

I'm sorry, let me fix that